command-expansion-server/src/app.ts

```diff
@@ -1,6 +1,9 @@
+import util from "util";
+
 import express, { Application, Request, Response, NextFunction } from "express";
 import bodyParser from "body-parser";
 import { GraphQLClient } from "graphql-request";
+import Ajv from "ajv";
```

Comment:
I think we can remove this and the validation call below + schema file as we have the graphql api doing type validation for us. I know @patkenneally and @Twisol had thoughts on that one though that may differ from my own, so I'll let them weigh in.

Reply:
Removed this. If we want server-side validation, we can add it in the future more comprehensively to all the endpoints.